singularity-eos/eos/eos_spiner.hpp

```diff
@@ -873,12 +873,12 @@ inline void SpinerEOSDependsRhoT::fixBulkModulus_() {
       Real DPDR = dPdRho_.interpToReal(lRho, lT);
       Real DPDE = dPdE_.interpToReal(lRho, lT);
       Real DEDR = dEdRho_.interpToReal(lRho, lT);
-      Real DTDE = dTdE_.interpToReal(lRho, lT);
+      Real DPDR_T = DPDR + DPDE * DEDR;
```

Comment:
while we're here would it make sense to explicitly change the variable names to make sure they include what should be held constant? In other words I'd advocate for `DPDR` being changed to `DPDR_E`.

Otherwise it's difficult to tell what exactly is being done here.